src/screens/LoginPage/LoginPage.tsx

```diff
@@ -57,6 +58,53 @@ function loginPage(): JSX.Element {
   const [showPassword, setShowPassword] = useState<boolean>(false);
   const [showConfirmPassword, setShowConfirmPassword] =
     useState<boolean>(false);
+  const [showLowercaseCharAlert, setShowLowercaseCharAlert] =
```

Comment:
Create a type in this file itself for PasswordValidation object, and club all these vairables into one

```ts
const [showAlert, setShowAlert] = useState<PasswordValidation> = ({
 lowercaseChar: true,
 uppercaseChar: true,
 numericValue: true,
 specialChar: true
})    
```

Reply:
Working on it

Reply:
@rishav-jha-mech 
Refactored the code as suggested